pyproject.toml

```diff
@@ -1,3 +1,6 @@
 [build-system]
-requires = ["setuptools", "cython", "versioneer"]
+requires = ["setuptools<72", "cython", "versioneer"]
```

Comment:
Sorry if this has already been covered - but why the new version bound?

Reply:
It was covered in one of the other unresolved comments. I'll have a closer look at that though, and will push a tweak - it's the last thing that really needs a change before merging.